src/database/relationships/index.js

```diff
@@ -14,6 +14,19 @@ export const associate = () => {
     otherKey: "userId",
   });
 
+  DB.Project.hasMany(DB.UserProject, {
+    onDelete: "CASCADE",
+    foreignKey: "projectId",
+  });
```

Comment:
It is a good practice to add the "as" keyword to provide meaningful aliases for the associations.

Hint: 
```javascript
DB.Project.hasMany(DB.UserProject, {
  onDelete: "CASCADE",
  foreignKey: "projectId",
  as: "userProjects", // Alias for the association
});
```